### ISSUE TITLE
Train Script

### ISSUE THREAD

**Issue by AndrewBoessen** [AndrewBoessen/PerfectRep#32]
Script to run pretraining on model. Load data from h3.6m and optional fit3d. Add arg parser for setting hyperparams. Add functionality for saving and loading checkpoint. Evaluate accuracy of model.
